src/test/kotlin/net/atos/client/sd/model/SdFixtures.kt

```diff
@@ -1,7 +1,134 @@
 package net.atos.client.sd.model
 
+import net.atos.client.sd.model.templates.DocumentsStructure
+import net.atos.client.sd.model.templates.GroupsAccess
+import net.atos.client.sd.model.templates.HeadersStructure
+import net.atos.client.sd.model.templates.Template
+import net.atos.client.sd.model.templates.TemplateGroup
+import net.atos.client.sd.model.templates.TemplatesResponse
+import net.atos.client.sd.model.templates.TemplatesStructure
+import net.atos.client.sd.model.templates.User
+import net.atos.client.sd.model.templates.UserGroup
+import net.atos.client.sd.model.templates.UsersStructure
+import net.atos.client.sd.model.wizard.WizardResponse
+import java.util.UUID
+
 fun createWizardResponse(
     ticket: String = "dummyTicket",
 ) = WizardResponse().apply {
     this.ticket = ticket
 }
+
+fun createUserGroup(name: String) = UserGroup().apply {
+    this.groupsAccess = GroupsAccess().apply {
+        this.templateGroups = listOf(
+            TemplateGroup().apply {
+                this.id = UUID.randomUUID().toString()
+                this.name = "Dimpact"
+                this.allDescendants = true
+            }
+        )
+        this.headerGroups = emptyList()
+    }
+    this.userGroups = emptyList()
+    this.users = listOf(
+        User().apply {
+            this.id = UUID.randomUUID().toString()
+            this.name = "zaakafhandelcomponent"
+        }
+    )
+    this.accessible = true
+    this.id = UUID.randomUUID().toString()
+    this.name = name
+}
+
+fun createTemplate(templateName: String) = Template().apply {
+    this.id = UUID.randomUUID().toString()
+    this.name = templateName
+    this.favorite = false
+}
+
+fun createTemplates() = listOf(
```

Comment:
Where did you get this list from? These names don't make sense to me in a test context. I'd only use 1 variant instead of both 'nieuw' and 'oud' variants for the templates. Unless there's a good reason to have them.

Reply:
That's the actual SD response atm. 
Wanted to make sure the current structure and types definitions we have can handle it. Had to adjust `TemplateGroup` and make some fields optional because of this.